beetsplug/mbcollection.py

```diff
@@ -28,8 +28,6 @@
 
 SUBMISSION_CHUNK_SIZE = 200
 UUID_REGEX = r'^[a-f0-9]{8}(-[a-f0-9]{4}){3}-[a-f0-9]{12}$'
-AUTH_ERRORS = (TokenRequestDenied, TokenMissing, VerifierMissing)
-
```

Comment:
I was searching for `rauth` description of exception names, but with no luck. The project's documentation isn't very good. Probably gonna switch to `requests_oauthlib`.